src/OpenLoco/src/Objects/ObjectIndex.cpp

```diff
@@ -80,7 +80,8 @@ namespace OpenLoco::ObjectManager
             currentState.dateHash = Numerics::ror(currentState.dateHash, 5);
             currentState.totalFileSize += file.file_size();
         }
-        currentState.numObjects |= (1 << 24);
+        // NB: flag 24 is original; flag 25 was introduced by OpenLoco for vehicle subtype purposes
```

Comment:
I think we could just make this a version byte.

Reply:
Agreed, considering the code gets rid of the upper 8 bits everywhere. How best to go about this? Just shift in a byte constant to the upper 8 bits?